Team8923_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team8923_CENTERSTAGE/BaseTeleOp.java

```diff
@@ -1,8 +1,68 @@
 package org.firstinspires.ftc.team8923_CENTERSTAGE;
 
-
 abstract public class BaseTeleOp extends BaseOpMode {
-    // add mechanism speeds and drive methods when mechanisms is ready
-    double slidesSpeed = 0.6;
-    double releaseSpeed = 0.9;
+
+    private boolean flipGondola = false;
```

Comment:
It is not! Same with pressToggle, because we were originally going to use the Toggle class for one of our controls but ended up changing them to something else and forgot about it (sorry!), so I think I'm just going to delete the Toggle class